Game/Resources/Opening Scene/Opening Scene.json

```diff
@@ -1384,6 +1384,117 @@
 			"scene" : false,
 			"static" : false,
 			"uid" : 1513071370
+		},
+		{
+			"Material" : 
+			{
+				"albedo" : "Resources/NewAssets/Textures/Rock/rock_albedo",
+				"metallic" : "",
```

Comment:
Missing texture here and for roughness. This has occurred in multiple PRs by various authors. Is it a bug with saving material in the editor?

Reply:
Using default texture.